website/docs/Pipelines/snM3C/README.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
+slug: /Pipelines/snM3C/README
+---
+# snM3C Overview
+
+| Pipeline Version | Date Updated | Documentation Authors | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [snM3C_v1.0.0](https://github.com/yourusername/snM3C/releases) | August, 2023 | [Kaylee Mathews](mailto:warp-pipelines-help@broadinsitute.org) | Please file GitHub issues in the [WARP repository](https://github.com/broadinstitute/warp/issues) |
+
+
+## Introduction to snM3C
+
+The snM3C workflow is a cloud-based computational workflow for processing single-nucleus methylome and chromatin contact (snM3C) sequencing data. The workflow is designed to demultiplex raw sequencing reads, align them, call chromatin contacts, and generate summary metrics. It is developed in collaboration with the laboratory of Hanqing Liu and Joseph Ecker. For more information about the workflow, please see the [YAP documentation](https://hq-1.gitbook.io/mc/).
+
+## Set-up
+
+### Installation
+
+To use the latest release of the snM3C pipeline, visit the [WARP releases page](https://github.com/broadinstitute/warp/releases) and download the desired version.
+
+<!--- Add a comment about running an old version of the workflow --->
```

Comment:
I'm assuming this is a comment for future us and won't render in the meantime, is that right?

Reply:
Nope, it won't render on the doc site. It should be ok.